cfme/infrastructure/virtual_machines.py

```diff
@@ -390,6 +396,68 @@ def current_snapshot_description(self):
             "/td[not(contains(@class, 'key'))]"])
         return sel.text(l).strip()
 
+    def equal_drift_results(self, row_text, section, *indexes):
+        """ Compares drift analysis results of a row specified by it's title text
+
+        Args:
+            row_text: Title text of the row to compare
+            section: Accordion section where the change happened; this section must be activated
+            indexes: Indexes of results to compare starting with 0 for first row (latest result).
+                     Compares all available drifts, if left empty (default).
+
+        Note:
+            There have to be at least 2 drift results available for this to work.
```

Comment:
Drift is created after SSA for the instance if instance params have changed since then